light_curves/code/fluxconversions.py

```diff
@@ -1,6 +1,24 @@
 #need to convert those magnitudes into mJy to be consistent in data structure.
 #using zeropoints from here: https://wise2.ipac.caltech.edu/docs/release/allsky/expsup/sec4_4h.html
 def convert_WISEtoJanskies(mag, magerr, band):
+    """converts WISE telescope magnitudes into flux units of Jansies in mJy
+    
+    Parameters
+    ----------
+    mag : array-like
+        array of WISE magnitudes
+    magerr : array-like
+        array of WISE uncertainties on the magnitudes
+    band : str {'w1', 'w2'}
+        name of the WISE band to be converted
+        
+    Returns
+    -------
+    flux: array
```

Comment:
sphinx rendering nitpick, spaces are needed before and after the `:`, otherwise it won't boldface correctly. You did it right for the Parameters, as I see it only affects the Returns

Reply:
OTOH, we don't do sphinx API docs generation, so this is not an issue whatsoever